lib/trie/trie.go

```diff
@@ -515,6 +538,10 @@ func (t *Trie) insertInBranch(parentBranch *Node, key, value []byte) (
 // The keys are in hexadecimal little Endian encoding and the values
 // are hexadecimal encoded.
 func (t *Trie) LoadFromMap(data map[string]string) (err error) {
+	defer func() {
+		t.handleTrackedDeltas(err)
+	}()
+
```

Comment:
```suggestion
	defer t.handleTrackedDeltas(err)
```

Reply:
Actually this is different I think; the original case will capture the returned named `err` error, but the second will capture the original (nil) error...

See https://go.dev/play/p/q1E3YATJ7ir